packages/components/src/border-control/border-control/hook.ts

```diff
@@ -14,16 +14,12 @@ import { useCx } from '../../utils/hooks/use-cx';
 
 import type { Border, BorderControlProps } from '../types';
 
-const sanitizeBorder = ( border?: Border ) => {
```

Comment:
I refactored the logic of this `sanitizeBorder` function so it can be reused to determine the `isStyleSettable` boolean. I renamed it and inverted the logic to `isValidBorder` for better readability within the file.